.ruby-version

```diff
@@ -1 +1 @@
-2.3.1
+2.7.1
```

Comment:
You could set this to `3.0.0` now

Reply:
i could do that ! i will do that

Reply:
https://github.com/meetcleo/zero_downtime_migrations/pull/4